### DISCUSSION TITLE
r19base:/11.tabs

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#20] [General]
# tutorial/use/tabs

http://localhost:5173/tutorial/use/tabs

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得

**Comment by dlgchg** [comment]
我来了

**Comment by wctiger** [comment]
> “这里当我们切换点击时，会修改两个数据，但是最终只有一个 state 变化”

这里我们需要两个数据吗？这个`current`状态是不是可以通过计算得出 像这样 
```
const tabsWithCurrent = tabs.map((t,index) => ({...t, current: index === current}));
```

**Reply by yangbo5207** [reply]
不用纠结这个实现，tab 数据的管理方式很多，我只是随便选择了一种
